.github/workflows/codeql.yml

```diff
@@ -0,0 +1,77 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
+    - cron: '34 22 * * 5'
```

Comment:
What is this schedule? Every 5th at 22:34?

Reply:
In the given cron schedule '34 22 * * 5', it means the CodeQL workflow will be triggered at:

34 minutes past the hour
22nd hour (10 PM)
Every day of the month
Every month
On the 5th day of the week (Friday)
So, the workflow will run every Friday at 10:34 PM.

Well, codeql has that, i did not change anything basically

Reply:
ok

Reply:
It did not mention time zone, then UTC maybe